packages/zowe-explorer-ftp-extension/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## TBD RELEASE
```

Comment:
I believe the changelog check is case sensitive, so this may need to be `TBD Release`